src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -112,6 +113,11 @@ public void callback(CallbackContext context) {
     UserIdentity userIdentity = userIdentityFactory.create(getUser(scribe, accessToken),
       settings.syncGroups() ? getTeams(scribe, accessToken) : null);
 
+    if (isOrganizationMembershipRequired() && !isOrganizationsMember(accessToken, userIdentity.getLogin())) {
+      throw new IllegalStateException(format("'%s' must be a member of at least one organization: '%s'",
```

Comment:
Sounds good.  IIRC I had to do this to support `5.4` which I know we're not targeting now.